Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -804,31 +746,44 @@ private static Result ValidateNodeDescription(string description, IChildNode nod
                 var type = command.GetCommandType();
                 if (type is ApplicationCommandType.ChatInput)
                 {
-                    return description.Length <= _maxCommandDescriptionLength
-                        ? Result.FromSuccess()
-                        : new UnsupportedFeatureError
-                        (
-                            $"A command description was too long (length {description.Length}, " +
-                            $"max {_maxCommandDescriptionLength}).",
-                            node
-                        );
+                    if (description.Length <= _maxCommandDescriptionLength)
+                    {
+                        return;
+                    }
+
+                    throw new UnsupportedFeatureException
+                    (
+                        $"A command description was too long (length {description.Length}, "
+                        + $"max {_maxCommandDescriptionLength}).",
+                        node
+                    );
+                }
+
+                if (description == string.Empty)
+                {
+                    return;
```

Comment:
I think for non-context commands, having an empty description should throw.

Reply:
You need to get more into detail here.
As this branch is actually enforcing an empty description for `USER` and `MESSAGE` commands.

Reply:
Discord requires the description to be empty (i.e, not present) for `USER` and `MESSAGE` commands. Can you clarify what you mean?

Reply:
AraHaan is actually referring to non-context commands (e.g. `CHAT_INPUT`).
According to Discord it must have a description with 1-100 characters, therefore empty string is actually invalid.

Reply:
Right, which is checked in the conditional above this one.. This check is only for commands that are not chat input.

Reply:
It checks if it is >100 characters but not if the description is empty.

Reply:
That is true; however, Remora.Commands does not permit an empty description, and will substitude it with `No description set.` if one is encountered. This code can never see an empty description unless it's been explicitly overridden by Remora.Discord.Commands, which is the case for USER and MESSAGE commands (line 516).